pages/contacts-call/[id]/index.js

```diff
@@ -40,27 +38,33 @@ const StyledRedButton = styled.div`
   align-items: center;
 `;
 
+const StyledBigPlaceholder = styled(GrayCirclePhotoPlaceholder)`
+  width: 9rem;
+  height: 9rem;
+  margin: 0;
+`;
+
+//----------------------------------------------- FUNCTION------------HERE
```

Comment:
I also do not think that you need this comment as your code structure is good! But that is up to you to decide and this will be my last time in this PR to mention this. :)